src/pyramid/view.py

```diff
@@ -23,6 +23,7 @@
 from pyramid.util import hide_attrs, reraise as reraise_
 
 _marker = object()
+LIFT = object()
```

Comment:
We have `pyramid.util.Sentinel` that should be used here if it's going to be exposed as an API.